test/index.js

```diff
@@ -50,15 +50,21 @@ test('read', function (t) {
 
 test('passing data', function (t) {
     // on reads it should be part of the URL
-    var xhr = sync('read', getStub(), {data: {a: 'a', one: 1}});
+    
+    //create a plain object, with no prototypal inheritance, like qs@v3 does 
```

Comment:
But why? Are we going to require that from users? I don't think so. It should remain a plain object.